framework/web/Request.php

```diff
@@ -234,6 +234,9 @@ class Request extends \yii\base\Request
      */
     public $ipHeaders = [
         'X-Forwarded-For',
+        'cf-connecting-ip',
+        'HTTP_X_REAL_IP',
+        'HTTP_X_FORWARDED_FOR',
```

Comment:
this is how they show up in `$_SERVER`, but the name of the header field is `X-Forwared-For` and  `X-Real-IP`.
Can you give a source or explaination for `X-Real-IP`?

Reply:
`HTTP_X_REAL_IP` and `HTTP_X_FORWARDED_FOR` headers from nginx, which in frontend
Not sure if header `HTTP_X_REAL_IP` or `X-REAL-IP` depends on `nginx` settings, in my server nginx+apache I get headers `HTTP_X_REAL_IP` and `HTTP_X_FORWARDED_FOR` with real user IP